datafusion/tests/sql.rs

```diff
@@ -3976,6 +3976,12 @@ async fn query_on_string_dictionary() -> Result<()> {
     let expected = vec![vec!["2"]];
     assert_eq!(expected, actual);
 
+    // aggregation min
```

Comment:
Since the changes are needed for min and max, I suggest also adding a test here for `max` to ensure it is covered